articles/credits-faq.md

```diff
@@ -0,0 +1,96 @@
+
+## How do I see my credits balance?
+You can see your current credit balance by going to the Overview blade on your Azure Quantum Workspace, and selecting the "Quota" tab. You will see a credits quota for each provider that you have credits with. You will see the total number of credits you have received, and may hover over the relevant graph to see the amount you have used towards your credits.
+
+You can also see your quotas by using the AZ CLI and the `az quantum workspace quotas` command, like in the example below. In this case, the `qgs` row shows that the account has a total of `8333334 qgs` with IonQ, of which `33334 qgs` have been used. The account also has `800 HQC` with Honeywell, of which `0 HQC` have been used.
+```bash
+$ az quantum workspace quotas -o table
+Dimension  Holds  Limit      Period    ProviderId  Scope         Utilization
+---------  -----  ---------  --------  ----------  ------------  -----------
+qgs        0.0    8333334.0  Infinite  ionq        Subscription  33334.0
+hqc        0.0    800.0      Infinite  honeywell   Subscription  0.0
+```
+
+Note that credits are shown in the quota units used by the provider, and not in the dollar value of the credits received. You may use the table below to understand the approximate conversion value between quota units and US dollar value:
+| Provider | Unit | Approximate $ Value (USD) | Documentation |
+|----------|------|---------------------------|---------------|
+| Honeywell | HQC (Honeywell Quantum Credit) | $1 USD ~= 0.08 HQC | [Learn more about how Honeywell Quantum Credits are calculated](xref:microsoft.quantum.providers.honeywell) |
+| IonQ | QGS (Qubit Gate-Shot) | $1 USD ~= 33,333.3 QGS | [Learn more about how IonQ Qubit-Gate-Shots are calculated](xref:microsoft.quantum.providers.ionq) |
+
+## I'm encountering a problem using my credits, what can I do?
+If you are having any problems with Azure Quantum pertaining to credits or otherwise, the best and fastest place to get support is by [filing a support ticket with Azure](https://azure.microsoft.com/support/create-ticket/).
+
+If you do not have an Azure support subscription or are unsure about how to file a ticket, you may reply to your approval notice email and one of our team members will get back to you.
+
+## Guide: Onboarding with Honeywell Credits
+Instructions for configuring your Azure Quantum Workspace to access Honeywell:
+
+1. [Open the Azure Quantum Workspace Create screen in the Azure Portal](https://portal.azure.com/#create/Microsoft.AzureQuantum)
+1. On the **Basics** tab while configuring your Workspace, ensure the selected **subscription** matches the subscription you supplied in your onboarding information. The subscription information is also copied in the onboarding mail you received.
```

Comment:
dropdown will show subscription name, but in form we are asking subscription id. Can we add a step in between to show them how to find the the name?

Reply:
Good idea, I've added this in the latest rev.